core-bundle/src/DependencyInjection/Compiler/RegisterPagesPass.php

```diff
@@ -62,6 +64,10 @@ protected function registerPages(ContainerBuilder $container): void
                 $class = $definition->getClass();
                 $type = $this->getPageType($class, $attributes);
 
+                if (\strlen($type) > self::MAX_PAGE_TYPE_LENGTH) {
```

Comment:
Shouldn't this consider the actual length of the field instead of a constant? Now, there is no way for me to increase the  length (SQL definition) if I run into the error.

Reply:
Hm, I don't know, can we evaluate this during the compiler pass?

Reply:
not really, you can't initialize the framework in that state. As I said before, I don't think that's necessary either. If someone **really** runs into that issue we can still increase it for everyone in Contao.